tests/Application/.env

```diff
@@ -26,5 +26,5 @@ SYLIUS_MAILER_PASSWORD=~
 SYLIUS_MAILER_SEND_FROM=your@mail.com
 ###< symfony/swiftmailer-bundle ###
 
-RECAPTCHA_SITE_KEY=6LfzPccUAAAAAEz_CC6E2H8BMsxPkiil1mak7QMB
-RECAPTCHA_SECRET_KEY=6LfzPccUAAAAAL1lsyGhLQ68gG0B4opbRTKTGi-p
+RECAPTCHA_SITE_KEY=6LdZcMcUAAAAANZggR_m4eIYriN051sFdiUkv1kZ
```

Comment:
Add this to the readme, so that the user knows what variables are

Reply:
this variables are temporary, they will be move in a yaml file at the next issue (https://mangoweb.org/apps/sylius-test-plugin/issues/20)

Reply:
https://github.com/mangoweb-sylius/SyliusContactFormPlugin/pull/6

Reply:
What does it mean?

Reply:
DONE